install_msw.sh

```diff
@@ -287,7 +304,7 @@ EOF
 # therefore, symlink the directories lmi uses as described in
 # 'README.schroot'.
 
-if [ "Cygwin" != "$platform" ]
+if [ "Cygwin" != "$platform" ] && [ "WSL" != "$platform" ]
```

Comment:
```suggestion
if [ "Cygwin" != "$platform" -a "WSL" != "$platform" ]
```

Reply:
`sygcheck` says:
```
SC2166: Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.
```

Reply:
I had no idea... [The rationale](https://github.com/koalaman/shellcheck/wiki/SC2166) seems pretty flimsy to me, but it's better to avoid the warning, of course.